clientv3/op.go

```diff
@@ -111,6 +134,22 @@ func OpPut(key, val string, opts ...OpOption) Op {
 	return ret
 }
 
+func opWatch(key string, opts ...OpOption) Op {
+	ret := Op{t: tRange, key: []byte(key)}
+	ret.applyOpts(opts)
+	switch {
```

Comment:
can you pull `getPrefix([]byte) []byte` out of `WithPrefix()` so this can check
`case ret.end != nil && reflect.DeepEqual(end, getPrefix(ret.key)): panic("only supports single keys or prefixes")`?